src/app/pages/plugins/plugin-add/plugin-add.component.ts

```diff
@@ -322,7 +318,7 @@ export class PluginAddComponent implements OnInit {
     }
 
     for (let i in value) {
-      if (value.hasOwnProperty(i) && i != 'https') {
+      if (value.hasOwnProperty(i) && i != 'jail_name') {
```

Comment:
`!==` for string comparison is probably the safest bet. avoids implicit type conversions.